lib/bpf/network.go

```diff
@@ -92,6 +109,36 @@ type rawConn6Event struct {
 
 	// Command is name of the executable making the connection.
 	Command [CommMax]byte
+
+	// SockType is the socket type, either SOCK_STREAM or SOCK_DGRAM.
+	SockType uint16
+
+	// SockInode is the inode backing the socket.
+	// Used to identify repeated sends by the same socket.
+	SockInode uint64
+}
+
+func (e *rawConn6Event) SrcIP() net.IP {
+	return ipv6HostToIP(e.SrcAddr)
+}
+
+func (e *rawConn6Event) DstIP() net.IP {
+	return ipv6HostToIP(e.DstAddr)
+}
+
+func ipv4HostToIP(addr uint32) net.IP {
```

Comment:
This and ipv6HostToIP are just moves. At some point I was using the net.IPs in multiple places; not anymore, but I decided to keep the refactor.